src/frontend/components/SøknadsSteg/VelgBarn/useVelgBarn.tsx

```diff
@@ -14,23 +16,50 @@ export const useVelgBarn = (): {
     skjema: ISkjema<IVelgBarnFeltTyper, string>;
     validerFelterOgVisFeilmelding: () => boolean;
     valideringErOk: () => boolean;
+    oppdaterSøknad: () => void;
+    håndterVelgBarnToggle: (barn: IBarn, erMedISøknad: boolean) => void;
+    barnSomSkalVæreMed: IBarn[];
 } => {
-    const { søknad } = useApp();
-    const { barnInkludertISøknaden: barnMedISøknad } = søknad;
+    const { søknad, settSøknad } = useApp();
+    const { barnInkludertISøknaden } = søknad;
+    const { settBarnForRoutes } = useRoutes();
+    const [barnSomSkalVæreMed, settBarnSomSkalVæreMed] = useState<IBarn[]>(barnInkludertISøknaden);
 
-    const barnMedISøknadFelt = useFelt<IBarn[]>({
-        verdi: barnMedISøknad,
+    useEffect(() => {
+        settBarnForRoutes(barnSomSkalVæreMed);
+    }, [barnSomSkalVæreMed]);
```

Comment:
Var dette kosher, å ha et array som dependency? Det var greit så lenge antallet dependencies var likt eller?

Reply:
Ja jeg tror det var antallet dependencies. Den klager ikke i consollen i allefall slik som den andre feilen vi hadde. Har testet med to barn.